alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/healthprocessor/plugins/ContentValidationHealthProcessorPlugin.java

```diff
@@ -72,7 +73,7 @@ protected NodeHealthReport process(NodeRef nodeRef) {
         }
 
         boolean nodeHasContent = false;
-        Set<QName> failures = new HashSet<>();
+        Set<Pair<QName, String>> failedPropertiesWithContentUrl = new HashSet<>();
```

Comment:
I would consider to just use a Map instead.

Is there an intention to transform the Pair into its own POJO that might be extended later? 
In that case, I can see why to keep the Set

Reply:
Very good point